python/cudf/cudf/dataframe/string.py

```diff
@@ -795,6 +795,16 @@ def is_monotonic_decreasing(self):
             ).all()
         return self._is_monotonic_decreasing
 
+    def value_counts(self, method="sort"):
+        if method != "sort":
+            msg = "non sort based value_count() not implemented yet"
+            raise NotImplementedError(msg)
```

Comment:
Lets just remove the method parameter from this function. It's going to use a hash based approach under the hood in this case but it's just confusing.